sdk/attestation/azure-security-attestation/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0 (2022-07-05)
```

Comment:
Are we shipping this on `2022-07-07`? We might want to update the date then.

Reply:
This is the date that the prepare_release.ps1 script chose. I assumed that this was the actual date of the release.

 I was planning on releasing it today, but I can hold off if necessary.